public/coding/JavaScript/JsFunctions/runningPaceCalculator/runningPaceCalculator-Practice.js

```diff
@@ -0,0 +1,22 @@
+function runningPaceCalculator(distanceMiles, timeHours, timeMinutes) {
+  /*
+   * In this challenge you will calculate the running pace per mile given the following parameters:
+   * 1) the race distance in miles, 2) the time in hours, and 3) time in minutes to complete the race
+   * The pace per mile should be out put in the HH:MM format. If a 13.1 mile race takes 1:45 to complete, the pace per mile function should output 8:35
```

Comment:
I would add a brief sentence that explains what pace mean. 

`Running pace is a term used to represent how long it takes a runner to run a mile` 

You can calculate the running pace by dividing the total time by total distance run.